windows/sound.cpp

```diff
@@ -864,240 +889,91 @@ void CSound::bufferSwitch ( long index, ASIOBool )
         {
             const int iSelCH = pSound->lNumInChan + pSound->vSelectedOutputChannels[i];
 
+#define CSOUND_PUT_DATA(format,endian,factor)                           \

```

Comment:
See above comment regarding the function defines.

Reply:
I know. Either you repeat that piece of code a dozen times, or you use that macro. I see no simple way to resolve this? Do you?